tools/airgap-install/README.md

```diff
@@ -17,16 +17,28 @@ To run the utility, type
 ./get-image-list <version>
 ```
 
-where **version** is the Gloo Mesh Enterprise version (e.g. 1.1.6).
+where **version** is the Gloo Platform version (e.g. 1.1.6).
 
 This will print the list of images to stdout.
 
 ## Options
-`-p | --pull` - pull the images locally.
+
+- `-ih | --istio-hub`         - Istio hub (Defaults to us-docker.pkg.dev/gloo-mesh/istio-workshops if not specified)
+- `-iv | --istio-ver`         - Istio version (Defaults to 1.20.2 if not specified)
```

Comment:
Yes, it's set to this line in the script, I believe, so whenever that changes, this will also have to be changed: https://github.com/solo-io/solo-cop/pull/126/files#diff-6ae9c2eb60f4ada224553d8ea2a932e52e2412a0996552f21055ddf420d735d0R12